netutils/netinit/Kconfig

```diff
@@ -103,6 +103,18 @@ config NETINIT_LOSS_POLL_RATE
 		The network monitor will check the PHY link state every
 		NETINIT_LOSS_POLL_RATE seconds for link loss.
 
+config NETINIT_FALLBACK
+	int "The number of failed DHCP attempts to fall back to using static settings"
+	default 5
+	depends on !ARCH_PHY_INTERRUPT && ARCH_PHY_POLLED && BOARDCTL_NETCONF
```

Comment:
remove BOARDCTL_NETCONF, why the fallback can't work with ARCH_PHY_INTERRUPT?

Reply:
@xiaoxiang781216 - Thank you I missed the one. I think it need to be FSUTIL_IPCFG to work. This is what provides the static IP. 

As far as ARCH_PHY_POLLED.  I do not have HW with the IRQ connected. The timing on the fall back is condition by the ARCH_PHY_POLLED, so I am not sure how it will work.

Do you have HW with the IRQ connected? Can you test it?

Reply:
done in 664208ece0b24ac2b219694111fc1847dcadc444 fixup